Zivid/ReadZDF/ReadZDF.cpp

```diff
@@ -29,6 +29,30 @@ int main()
 
 		std::cout << "Running the visualizer. Blocking until the window closes" << std::endl;
 		vis.run();
+
+
+		// Extract point cloud from the frame
```

Comment:
For consistency with other samples, use 'ing' tense in comments, i.e. // Extracting point cloud from the frame